resources/assets/js/components/FindSkillsModal.tsx

```diff
@@ -476,7 +515,7 @@ const FindSkillsModal: React.FunctionComponent<FindSkillsModalProps> = ({
               data-h2-padding="b(rl, 2) b(tb, .5)"
               data-h2-bg-color="b(white, 1)"
               onClick={() => {
-                setSkillsResults([]);
+                setNewSkills([]);
```

Comment:
I just noticed that this will trigger if you click Cancel to close the dialog, but _not_ if you press Esc. Its not a big deal in this case, but its something we need to be aware of when using this version of the Dialog component.

Reply:
So, I changed it to set the new skills array to zero `setNewSkills([])` only after submission. This should trigger a re-render and if they open the add skills modal again it will set those skills they just added to "Added already". Then I removed any events if they hit cancel, so if they hit cancel and open it again they will pick up wear they left off.